### PR TITLE
Handle videos with data codec_type by ignoring it. Handle videos without audio.

### PR DESCRIPTION
**Pull request by rabzu**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name := "media4s"
 organization := "org.matthicks"
 version := "1.0.14"
-scalaVersion := "2.13.0"
```
Comment (darkfrog26) [inline build.sbt:4]: Remove this. I don't want the default Scala version to change.

File: src/main/scala/org/matthicks/media4s/video/info/MediaInfo.scala [hunk 2]
```diff
@@ -35,21 +37,28 @@ case class MediaInfo(streams: List[MediaStream], format: MediaFormat) {
...
-  def audio: AudioInfo = audios.head
+  def audio: Option[AudioInfo] = audios.headOption
   def frames: Int = videos.headOption.map(_.fps * duration).map(_.toInt).getOrElse(0)
 
   def hasVideo: Boolean = videos.nonEmpty
   def hasAudio: Boolean = audios.nonEmpty
 
   override def toString: String = {
-    s"MediaInfo(duration: $duration, start: $start, video: $video, audio: $audio, frames: $frames)"
+    val audioString = audio.fold("No Audio")(audio => s"$audio")
```
Comment (darkfrog26) [inline src/main/scala/org/matthicks/media4s/video/info/MediaInfo.scala:47]: Perhaps:
`val audioString = audio.map(_.toString).getOrElse("No Audio")`
Might be a little clearer?

**Comment by darkfrog26** [comment]
Apart from the couple of suggestions I made, this looks good.

File: src/main/scala/org/matthicks/media4s/video/info/MediaInfo.scala (hunk 2 above)
Comment (rabzu) [reply src/main/scala/org/matthicks/media4s/video/info/MediaInfo.scala:47]: Hi, 
Thanks. Is the code unclear or the comment?

Comment (darkfrog26) [reply src/main/scala/org/matthicks/media4s/video/info/MediaInfo.scala:47]: Personally, I think the code is unclear.  If you disagree, I won't hold-back the merge for this.

File: build.sbt (hunk 1 above)
Comment (rabzu) [reply build.sbt:4]: Sure

File: src/main/scala/org/matthicks/media4s/video/info/MediaInfo.scala (hunk 2 above)
Comment (rabzu) [reply src/main/scala/org/matthicks/media4s/video/info/MediaInfo.scala:47]: Sure, I can make this more clear.

**Pull request closed without merging**